libraries/Microsoft.Bot.Builder.Dialogs.Debugging/vscode-dialog-debugger/package.json

```diff
@@ -41,7 +41,15 @@
         "label": "Bot Framework Dialog Debugger",
         "languages": [
           "dialog"
-        ]
+        ],
+        "initialConfigurations": [
```

Comment:
>initialConfigurations [](start = 9, length = 21)

should allow adding a new launch configuration for other declarative bot json directories